### PR TITLE
Edited getting started content into README format

### PR DESCRIPTION
**Pull request by adamd**
We're working with @jdanielmyers to prepare developer content for Snowflake. If you have questions, feel free to add them as comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,143 @@
...
+  
+    
+For advanced features: Auto-generate Data Types
+
+* usage on all procedures
+* usage on future procedures
+* usage on all functions
+* usage on future functions
+
+One approach for handling these privileges is to create a new role with the above Schema and Database privileges and assigned that role existing role with the Table level privileges.
```
Comment (adamd) [inline README.md:67]: We weren't able to figure out what this is suggesting. There _might_ be a typo in this sentence, but we aren't sure what edit it needs.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,143 @@
...
+
+Repeat as-needed to gather data from Snowflake into your Excel sheet.
+
+### Write Data to Snowflake
+
+Once the data is returned you can update or insert rows and columns within Excel.
+
+To propagate those changes to Snowflake press the “Upload” button.  Here you can enter the worksheet the data is being sourced from and define the table the data is being written to.
+
+Another important option is defining the columns that will represent the keys in order to do the update logic needed if the user updates or adds a new column in the Excel sheet.  This can be done by entering the column position ex. A,B,C.  So the first 3 columns in the excel sheet and target table will be the key
```
Comment (adamd) [inline README.md:128]: Similar, we aren't sure what's being communicated here, but don't want to edit out important information!

File: README.md (hunk 1 above)
Comment (ssegal100) [reply README.md:67]: Thanks Adam. Would you like to get together to discuss?

Comment (ssegal100) [reply README.md:67]: Ah yes, I see the typo. I'll fix it and hopefully it will make more sense.

**Pull request merged**